File: homework-oo/03/04-05/Person.java
public class Person {
    private String name;

    public Person(String name) {
        setName(name);
    }

    public void setName(String name) {
        this.name = name;
    }

    public String getName() {
        return this.name;
    }

    public boolean equals(Person person) {
        return person.name.equals(this.name);
    }
}

File: homework-mobile/03/01/Person.kt

class Person (var name: String)

class Circle (var radius: Int? = null, var color: String? = null) {

    constructor(radius: Int?) : this(radius, null) {

    }
    fun printSurfaceArea() {
        val area : Double = Math.PI * (radius ?: 0) * (radius ?: 0)

        println(area)
    }

    fun attributesToString() {
        val output : String = "radius = ${radius ?: "null"}, color = ${color ?: "null"}"
        println(output)
    }
}

fun main() {
    var jack = Person("jack smith")

    println(jack.name)

    val c = Circle(5,)
    val c2 = Circle()

    println(c.radius)

    println(c2.radius)

    c.printSurfaceArea()

    println(c.attributesToString())

    c2.printSurfaceArea()

    println(c2.attributesToString())

}